src/com/projectkorra/projectkorra/PKListener.java

```diff
@@ -257,8 +257,9 @@ public void onBlockFlowTo(BlockFromToEvent event) {
 			}
 
 			if (!event.isCancelled()) {
-				if (TempBlock.isTempBlock(toblock) && Illumination.getBlocks().containsKey(TempBlock.get(toblock))) {
-					toblock.setType(Material.AIR);
+				if (Illumination.isIlluminationTorch(block)) {

```

Comment:
Uh, the variable `block` does not exist in this scope. Are you using a compiler to check these commits?

Reply:
Alright, fixed it in a new commit... and no, usually I don't need to xD... kind of looks like it with that occurring twice in a row though.

Reply:
So you haven't been testing your changes in-game or in a compiler?

Reply:
Basically all of my commits so far haven't been new mechanics- they've been fixes for errors in existing code (or code not working as intended), and I have done my best to keep most of the code the same so far (and make sure that the rest of the plugin works around it). So long as I only change one class but keep the existing functionality the same, I don't see that I need to test the entire plugin, unless I do a larger amount of changes.

I found a significant number of these errors (such as checking if a collection contains the wrong type of object) by using an IDE, and just cloning the project into my editor with Git... which means that all those errors were made by committing directly without testing in an IDE.

I get your point, and I certainly don't want to add more bugs to the plugin. Most of the time I am fairly careful, and I test significant changes.

Reply:
I think what I'll do is test in batches, that way I'm not doing individual tests for things like the nether wart typo. I don't think all changes like that need testing... but that helps accomplish it.